---a/net/minecraft/command/ICommand.java
+++b/net/minecraft/command/ICommand.java
@@ -1,20 +1,22 @@
 package net.minecraft.command;
 
 import java.util.List;
+
 import net.minecraft.util.BlockPos;
 
-public interface ICommand extends Comparable
-{
+public interface ICommand extends Comparable {
+
     String getName();
 
     String getCommandUsage(ICommandSender sender);
 
     List getAliases();
 
-    void execute(ICommandSender sender, String[] args) throws CommandException;
+    void execute(ICommandSender sender, String[] args) throws CommandException, WrongUsageException; // CraftBukkit - Add exception; // SpigotMCP NOTE -- WrongUsageException uneccicary
 
     boolean canCommandSenderUse(ICommandSender sender);
 
     List addTabCompletionOptions(ICommandSender sender, String[] args, BlockPos pos);
 
     boolean isUsernameIndex(String[] args, int index);
-}
+}
